ddtrace/llmobs/_llmobs.py

```diff
@@ -347,8 +348,18 @@ def flush(cls) -> None:
 
     @staticmethod
     def _patch_integrations() -> None:
-        """Patch LLM integrations."""
-        patch(**{integration: True for integration in SUPPORTED_LLMOBS_INTEGRATIONS.values()})  # type: ignore[arg-type]
+        """Patch LLM integrations. Ensure that we do not ignore DD_PATCH_MODULES or DD_TRACE_%_ENABLED settings."""
+        integrations_to_patch = {integration: True for integration in SUPPORTED_LLMOBS_INTEGRATIONS.values()}
+        dd_patch_modules = os.getenv("DD_PATCH_MODULES")
+        dd_patch_modules_to_str = parse_tags_str(dd_patch_modules)
+        integrations_to_patch.update(
+            {k: asbool(v) for k, v in dd_patch_modules_to_str.items() if k in SUPPORTED_LLMOBS_INTEGRATIONS.values()}
+        )
+        for module, _ in integrations_to_patch.items():
+            env_var = "DD_TRACE_%s_ENABLED" % module.upper()
+            if env_var in os.environ:
+                integrations_to_patch[module] = asbool(os.environ[env_var])
```

Comment:
probably a very unlikely scenario, but (more for my own understanding) this means that we discard whatever was set from `DD_PATCH_MODULES` in some cases, ie
```
DD_PATCH_MODULES=openai:False
DD_TRACE_OPENAI_ENABLED=True
```
we would enable OpenAI, even though it is set to `False` from `DD_PATCH_MODULES`. is that the same behavior with the current patching logic in `patch_all` (we merge the values instead of going with `False` if even one is `False`)? again probably a very unlikely case lol (i don't know why someone would have two different values explicitly), but I'm curious if we're consistent with that behavior

Reply:
In the Python tracer, `DD_PATCH_MODULES` overrides `DD_TRACE_%_ENABLED` (see [preload](https://github.com/DataDog/dd-trace-py/blob/e04c8acf72186e63998cb4c97df0d3708aaeb15d/ddtrace/bootstrap/preload.py#L114-L117) and [patch_all](https://github.com/DataDog/dd-trace-py/blob/e04c8acf72186e63998cb4c97df0d3708aaeb15d/ddtrace/_monkey.py#L230)). I'll follow the same pattern to avoid inconsistencies. Thanks for the call out!